pkgs/games/netris/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchFromGitHub, ncurses }:
+
+stdenv.mkDerivation {
+  name = "netris";
+  version = "0.52";
+
+  src = fetchFromGitHub {
+    owner = "naclander";
+    repo = "netris";
+    rev = "6773c9b2d39a70481a5d6eb5368e9ced6229ad2b";
+    sha256 = "0gmxbpn50pnffidwjchkzph9rh2jm4wfq7hj8msp5vhdq5h0z9hm";
+  };
+
+  buildInputs = [
+    ncurses
+  ];
+
+  configurePhase = ''
```

Comment:
Its generally better to avoid replacing the `configurePhase`, this allows pre- and postConfigure to be easily injected later, try this:
```
  configureScript = "./Configure";
  dontAddPrefix = true;
```

Reply:
got it, thanks! :-)